package.json

```diff
@@ -41,7 +41,7 @@
     "tsup": "^8.2.4",
     "turbo": "^2.0.14",
     "typescript": "^5.5.4",
-    "vitest": "^2.0.5",
+    "vitest": "^2.1.2",
```

Comment:
There's bug in 2.0.5 that prevents the ! exclusion in the workspace config from working properly

https://github.com/vitest-dev/vitest/issues/6499